src/pages/ProfilePage/ProfileComponent/ListComponent.js

```diff
@@ -118,7 +118,7 @@ export default function ListComponent() {
     <div className={classes.root}>
       <List subheader={<Typography variant="h1">Activities</Typography>}>
         {ActivityItems.map((items) => (
-          <ListItem button key={items.id} onClick={items.id === 3 ? handelchange : items.onPress}>
+          <ListItem button key={items.id} onClick={items.id === 4 ? handelchange : items.onPress}>
```

Comment:
Thanks, @sakshi-choudhary  for PR actually it was not a wrong routing. I was thinking of making some changes to the profile section. I have left  few suggestions please have a look 

 1.  Instead of `Contests and Statistics Dashboard` make it `View Contests`
 2.  Add Expandable list to  `Classroom`  and add sub list `Student Dashboard`  and make it route to the Student Dashboard Screen. I think later we are going to add a `statistic component` to the Student Dashboard screen!

I think this change would be more clear and elements would be in right place :)

if @Abhishek-kumar09  agrees with the thought you can make changes =)

Reply:
okay,  Actually the page you made to submit assignments that is a classroom. So it should be linked to classroom menu.

Reply:
> okay, Actually the page you made to submit assignments that is a classroom. So it should be linked to the classroom menu.

Sure thing @sakshi-choudhary :)